eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/mutable/MultiReaderHashBag.java

```diff
@@ -481,34 +481,6 @@ public MutableShortBag collectShort(ShortFunction<? super T> shortFunction)
         }
     }
 
-    @Override
```

Comment:
I think these should be default methods in `MultiReaderBag`.

Reply:
@donraab why? To change the return type to MultiReaderBag? If so, that would be a good change, but one that has to happen in a major version. I think the minor version change looks good as-is, pulling these up to the MutableBag interface.

Reply:
Fair enough. I like the idea of returning MultiReaderBag though so access to the multi-reader specific API is retained.

Reply:
I started this change just now in https://github.com/eclipse/eclipse-collections/pull/879

Once this PR lands, I can add the new methods too.